language-server/dm/executionManager.mli

```diff
@@ -28,6 +28,7 @@ val is_remotely_executed : state -> sentence_id -> bool
 val get_proofview : state -> sentence_id -> Proof.data option
 val get_context : state -> sentence_id -> (Environ.env * Evd.evar_map) option
 val get_lemmas : (Environ.env * Evd.evar_map) -> completion_item list
+val get_location : (Environ.env * Evd.evar_map) -> string -> (string * ((int * int) * (int * int)) option) option
```

Comment:
Here the position tuple is `(int * int) * (int * int)`, but in DocumentManager it is `Position.t * Position.t`. Should they use a common import?